mfr/extensions/pdf/settings.py

```diff
@@ -0,0 +1,9 @@
+from mfr import settings
+
+
+config = settings.child('PDF_EXTENSION_CONFIG')
+
+EXPORT_TYPE = config.get('EXPORT_TYPE', 'pdf')
+EXPORT_MAXIMUM_SIZE = config.get('EXPORT_MAXIMUM_SIZE', '1200x1200')
+EXPORT_EXCLUSIONS = config.get('EXPORT_EXCLUSIONS', ['.pdf', ])
+EXPORT_MAX_PAGES = config.get('EXPORT_MAX_PAGES', 40)
```

Comment:
The above two config settings will have problems if set from the env.  Envvars are always strings; they won't be ints or lists.  I believe there are some helper methods in the config base class to cast these properly.